.ci/docker-steps.yml

```diff
@@ -77,6 +77,8 @@ steps:
     volumes: |
       $(build.sourcesDirectory):/src
       $(build.binariesDirectory):/build
+    envVars: |
+      R_LIBS_USER='/build'
```

Comment:
Without this, the package is installed to "/usr/local/lib/R", which is not a loaded docker volume.